### DISCUSSION TITLE
DApper - Retornar Somatorio de campo Total tipo double

### DISCUSSION THREAD

**Discussion by acostagit** [balta-io/forum#8] [Tira Dúvidas] [answered]
Como retornar o somatório de uma query, campo Total, double.?
No exemplo abaixo, o retorno ser´assim, mas preciso retornar a soma total, para a Tela do sistema.

Total
  10,50
150,00
200,00
250,00
----------------------

**Comment by andrebaltieri** [comment, marked as answer]
Executando um SUM na sua query, ou trazendo os dados e executando um SUM via LINQ

**Reply by acostagit** [reply]
Oberigado, mas como? Tem um exemplo?

**Reply by andrebaltieri** [reply]
@acostagit não tenho!

**Comment by acostagit** [comment]
Blza, que suporte.

Vou cancelar minha assinatura!

Vlws

On Thu, 22 Sep 2022 at 13:02 Andre Baltieri ***@***.***>
wrote:

> @acostagit <https://github.com/acostagit> não tenho!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/8#discussioncomment-3710800>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGHZH2KGO4XUQCTYRFEXVLV7R7HXANCNFSM6AAAAAAQL3AJPA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Obrigado!

Eng. Adriano A M Costa
"Viver é evoluir todo dia."
